packages/api/src/brevo-api/brevo-api-campaigns.service.ts

```diff
@@ -14,7 +15,11 @@ export class BrevoApiCampaignsService {
 
     constructor(@Inject(BREVO_MODULE_CONFIG) private readonly config: BrevoModuleConfig) {
```

Comment:
Not directly related to this PR but while we are at it: Why is `BREVO_MODULE_CONFIG` not provided by the `BrevoModule`? That would make more sense to me. It doesn't seem completely logical that I have to provide the config via the `ConfigModule` AND pass it to the `BrevoModule`